src/components/inscriptionPreview/index.js

```diff
@@ -0,0 +1,66 @@
+import { useState } from "react";
```

Comment:
@topether21 can u review this component? u initially created the first version of it.